### PR TITLE
feat: playground adds access to local components for development and debugging

### PR DESCRIPTION
**Pull request by baiwusanyu-c**
This pr mainly modifies the content of the playground, and supports the use of local component libraries for debugging.
1. Added several proxy files (such as onu-dev-proxy.ts), the playground will use the vue dependencies and onu-ui that will be accessed through them when running in the project
2. Write a few simple play debugging instructions, change the onu-ui code, and the playground can respond to updates
3. I'm sorry that I haven't solved the type error report, because I'm not a ts type expert, and I need some guidance to solve it perfectly.

others:
1. Edit the template file in the playground, the browser does not update the problem (see: [137](https://github.com/onu-ui/onu-ui/issues/137))
2. Obtaining the wrong version of onu-ui through CDN (see: [137](https://github.com/onu-ui/onu-ui/issues/137))

### CODE REVIEW

File: playground/vite.config.ts [hunk 1]
```diff
@@ -4,11 +4,15 @@ import UnoCSS from 'unocss/vite'
 import vue from '@vitejs/plugin-vue'
 import AutoImport from 'unplugin-auto-import/vite'
 import Components from 'unplugin-vue-components/vite'
-
+import execa from 'execa'
```
Comment (baiwusanyu-c) [inline playground/vite.config.ts:7]: Get the commit number and set it as a global variable

File: playground/src/utils/dependency.ts [hunk 2]
```diff
@@ -10,7 +11,15 @@ export const genCDNLink = (
   path = '',
   cdnUrl: string,
 ) => {
-  version = version ? `@${version}` : ''
+  if (IS_DEV && (version === `@${__COMMIT__}` || /vue/.test(pkg))) {
```
Comment (baiwusanyu-c) [inline playground/src/utils/dependency.ts:14]: If it is developed locally, use local vue dependencies and onu-ui paths

File: playground/playground.config.ts [hunk 3]
```diff
@@ -50,4 +50,19 @@ export default {
     skypack: 'https://cdn.skypack.dev/',
     jsdelivr: 'https://cdn.jsdelivr.net/npm/',
   },
+  devDeps: {
```
Comment (baiwusanyu-c) [inline playground/playground.config.ts:53]: Point to the proxy file, the address here can be optimized

File: package.json [hunk 4]
```diff
@@ -11,7 +11,9 @@
     "docs:build": "npm -C docs run build",
     "stub": "pnpm -r --parallel --filter=./packages/* run stub",
     "build": "rimraf packages/*/{dist,es,lib} && nr -r -F \"./packages/{preset,utils,components}\" build && nr volar:helper",
-    "play": "npm -C playground run dev",
+    "play": "run-s build play:run",
```
Comment (baiwusanyu-c) [inline package.json:14]: Execute build first and then execute dev and playground dev

File: playground/src/App.vue [hunk 5]
```diff
@@ -65,10 +67,11 @@ window.addEventListener(
 <template>
   <div class="comp-lib-play">
     <Header :store="store" />
+    <!-- store as any because store.compiler is shallowRef -->
     <Repl
       v-if="!loading"
       ref="repl"
-      :store="store"
+      :store="store as any"
```
Comment (yzh990918) [inline playground/src/App.vue:74]: 😄

**Review by yzh990918 — APPROVED** [review]
Thanks

**Pull request merged**